### DISCUSSION TITLE
Make choices collapsable.

### DISCUSSION THREAD

**Discussion by kentivar** [home-assistant/frontend#11468] [Automations / scripts / scenes]
When im making new automations with alot of choices it becomes cluttered really fast, its difficult to navigate and keep track of different choices in the automation. but if the choices had the abillity to be collapsed so i can "hide" them, that wold save us alot of scrolling :)

**Comment by spacegaier** [comment]
I think that is something that should go into the new redesign: https://github.com/home-assistant/frontend/discussions/10064
